.github/workflows/ci_cron_weekly.yml

```diff
@@ -104,7 +104,7 @@ jobs:
     # we include them just in the weekly cron. These also serve as a test
     # of using system libraries and using pytest directly.
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Maybe we should leave an inline comment there as well to link to upstream issues like
- https://github.com/tonistiigi/binfmt/issues/215